lib/db.js

```diff
@@ -1114,6 +1112,67 @@ module.exports = (
       })
   }
 
+  // Reduce redis memory usage by not encoding the keys. Store properties
+  // as fixed indices into arrays instead. Takes a session token object as
+  // its argument, returns an array.
+  function packTokensForRedis (tokens) {
+    return Object.keys(tokens).reduce((result, tokenId) => {
+      const unpackedToken = tokens[tokenId]
+
+      // Use `|| 0` to save space, because it's the shortest falsey value we can stringify
```

Comment:
Hrm...is there ever a risk of us wanting to store a `0` and distinguish it from `undefined`?  I guess it doesn't seem so with the current set of properties.

Reply:
Hmmm, I guess there is. I'll change it to leave falsey values alone.